src/gromacs/gpu_utils/gpu_utils.hip.cpp

```diff
@@ -90,7 +90,7 @@ bool isHostMemoryPinned(const void* h_ptr)
 //#if HIPRT_VERSION < 11 * 1000
 //            isPinned = true;
 //#else
-            isPinned = (memoryAttributes.memoryType == hipMemoryTypeHost);
+            isPinned = (memoryAttributes.type == hipMemoryTypeHost);
```

Comment:
```suggestion
#if HIP_VERSION_MAJOR > 5
            isPinned = (memoryAttributes.type == hipMemoryTypeHost);
#else
            isPinned = (memoryAttributes.memoryType == hipMemoryTypeHost);
#endif
```

Reply:
Sure we can take your suggested change. Though the member hipPointerAttribute_t.type has been existing since ROCm5.5.